src/components/tables/PoolsTable/TokenPills/TokenPills.vue

```diff
@@ -18,13 +16,15 @@
         :hasBalance="hasBalance(token.address)"
         :symbol="symbolFor(token)"
         :weight="weightFor(token)"
+        :isSelected="selectedTokens.includes(token.address)"
+      />
+      <HiddenTokensPills
+        v-if="hiddenTokens.length > 0"
+        :tokens="hiddenTokens"
+        :hasBalance="hasBalanceInHiddenTokens"
+        :isSelected="true"
```

Comment:
Hidden token pills are currently only showed for weighted pools - for stable we want to show all of them for now (it's up to 5 which fits nicely)